### DISCUSSION TITLE
WIN10，用不到lantern，提示“状态：无法将蓝灯设置为系统代理”

### DISCUSSION THREAD

**Discussion by ttccg** [getlantern/lantern-forum-cn#1053] [Q&A]
![微信图片_20241127182922](https://github.com/user-attachments/assets/a4dc94a6-edde-461c-893f-59ca3e3d40f9)

WIN10系统，用不到lantern，提示“状态：无法将蓝灯设置为系统代理”。

已经尝试过各种网上能找到的修复方法，但是全部都无法修复，已经感到心力交瘁。 T_T

还有就是按任务栏图标右键的“显示蓝灯”，不会在浏览器上打开蓝灯界面。

最后问一下蓝灯的端口，在settings.yaml看到有三个，addr, socksaddr, uiaddr:localhost，在win系统里的手动代理设置那里，应该填哪个？

我是付费专业版的用户。

请问怎样可以修复？

**Comment by MaxMKMax** [comment]
@ttccg 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

我们将非常感谢您。

**Comment by MaxMKMax** [comment]
@ttccg 
https://lantern.io/zh-Hans/faq
这个常见问题的连接或许也可以帮到您. 如果不是bug的话.

**Comment by MaxMKMax** [comment]
@ttccg 您好. 请问您是否还遇到这个问题? 您有查看您是最新版本吗? 请您更新最新版本.

**Comment by MaxMKMax** [comment]
@ttccg 如果您还遇到同样问题的话请您帮我在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by ttccg** [comment]
谢谢mike，目前还是修复不到，每次使用都是要手动设置，由于电脑比较多重要软件，不想重装系统，先将就着使用。再次感谢！

**Comment by greySkiesMarsh** [comment]
Hi @ttccg 

This is a very interesting issue and I'd love to present this to our engineers to fix for you. First, would you please help me out with getting some diagnostic information from your system? I've got some instructions here to guide you through the process. 

Windows

1.     Open File Explorer and navigate to Windows(C:) drive
2.     Select the Users folder
3.     Select the user account that installed Lantern
4.     Select the View button from the ribbon menu, mouse over the Show option, and select Hidden items. You should now see a folder named AppData
6.     Open the AppData folder, and continue to open Roaming > Lantern > logs
7.     Logs are contained in the text document logs
8.     Share this file in an email to [support@getlantern.org](mailto:support@getlantern.org)

**Comment by MaxMKMax** [comment]
@ttccg 这是一个非常有趣的问题，我很乐意将其呈现给我们的工程师来为您解决。首先，您能帮我从您的系统中获取一些诊断信息吗？我这里有一些说明来指导您完成整个过程。

请你跟着以上的步骤 ^

**Reply by ttccg** [reply]
不好意思，很久没上github。
刚刚已经按步骤发送文件了。

**Reply by ttccg** [reply]
1、WIN10系统，打开蓝灯后，如下图显示


![1](https://github.com/user-attachments/assets/78301a3b-f17d-4c70-913f-061c4cf18cc2)


================================

2、打开settings.yaml文件，找到IP和端口。

![image](https://github.com/user-attachments/assets/02b09045-5ca4-491c-b68c-feb519bec8c3)


=========================================

3、在系统代理设置里填入IP和端口，并保存。

![3](https://github.com/user-attachments/assets/9bccfdc2-c7a0-4099-b6fe-3c64a6e1991f)


====================

4、可以正常访问。但是依然无法打开lantern的主页。

![4](https://github.com/user-attachments/assets/2868b9a0-ee98-4937-a771-6a1e42f4e72e)


==========================

[lantern.log](https://github.com/user-attachments/files/19844032/lantern.log)

**Comment by MaxMKMax** [comment]
@ttccg 没问题. 谢谢您过了这么久还记得回来给我们提供资料. 我会帮您反馈给技术部门的. 

我们今天刚出了最新版本. 请您下载最新版本用用看. 谢谢! 
https://github.com/getlantern/lantern
8.2.0
